src/list_vault_jwt_expiration/list.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import logging
+import os
+
+import list_vault_jwt_expiration.context
+
+from list_vault_jwt_expiration.list import validate_vault_jwt_expiration
+from lib.logging import setup_logging
+from lib.validations import (
+  hostname_agrees_with_artsy_environment
+)
+
+
+def parse_args():
+  ''' parse command line args '''
+  parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    # TODO: figure out what this does
```

Comment:
which part? 😂

Reply:
😬 left over. Will remove. 

Also, I swear I wrote a PR description. Must have failed to save :(. Will update.